sal/class.json-api-site-base.php

```diff
@@ -507,4 +542,16 @@ function get_wordpress_version() {
 		global $wp_version;
 		return $wp_version;
 	}
+
+	protected function is_a8c_publication( $post_id ) {
```

Comment:
And don't forget to leave behind:

```php
abstract function is_a8c_publication( $post_id );
```

so that our subclasses are forced to conform to the interface definition.